guide/14-deep-learning/how-sequencetosequence-works.ipynb

```diff
@@ -0,0 +1,859 @@
+{
```

Comment:
<ol><li><strong><s>"sequence_translation"</s></strong><s>&nbsp;for SequenceToSequence model</s> - Available choice for <code>SequenceToSequence</code> model is <strong>"sequence_translation"</strong></li><li>Content of LICENSE has to be changed. The license file should contain attribution for United states only and not for other countries. This license file contain attribution for house addresses for 30 countries (it was created specifically for sample notebook for TextClassifier)</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/869/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>fixed</p>